packages/cli/src/commands/datasources/pull/index.ts

```diff
@@ -26,7 +29,14 @@ datasourcesCommand
 
     // Command options
     const { space, path } = datasourcesCommand.opts();
-    const { separateFiles, suffix, filename = 'datasources' } = options;
+    const {
+      separateFiles = false,
+      suffix,
+      filename = 'datasources',
```

Comment:
nitpick:

Since defaults were added at the commander level, you can remove them from the deconstructions. This probably applies to all the command files where defaults were added